### ISSUE TITLE
"The template deployment failed with error:" - Reader-scenario

### ISSUE THREAD

**Issue by villo123** [PacktPublishing/Penetration-Testing-Azure-for-Ethical-Hackers#4]
Hi.

I executed the "reader-scenario.ps1 script:

PS /home/azureadmin> ./reader-scenario.ps1
Deployment Started 02/22/2022 23:00:34
Please enter a password: *******
###########################################################################
# Creating new admin user **** in Azure AD #
###########################################################################

####################################################################
# Assigning the Reader role to **** #
####################################################################
{
....
}

However, I received the following message:

![reader-error](https://user-images.githubusercontent.com/8183363/155235633-f43c3913-28b6-43e1-8901-de61d7456911.png)

Could you help me, to resolve the issue.

Thank you.

Regards.

**Comment by villo123** [comment]
Hi.

I solved, doing this:

On your Pentest VM properties I changed the VCPU and RAM sizes from 4/16 to 2/7
![imagen](https://user-images.githubusercontent.com/8183363/156095255-2ad74f26-4ab1-416f-82df-04c6dd076794.png)

Then, on the Azure Cloud Shell I executed the reader-scenario-cleanup.ps1, finally I ran again the reader-scenario.ps1 script.

So,  the pentest-rg group was created succesfully with the required resources:

![imagen](https://user-images.githubusercontent.com/8183363/156096045-4d3d6595-8676-454d-ab68-85bea4dac6c4.png)

With this steps i could continue.

I hope that, help you.

Regards.
